src/components/fields/checkbox-array.js

```diff
@@ -83,12 +86,13 @@ export default createReactClass({
             <span key="spacer" renderWith={this.renderWith('InputSpacer')}>
               {' '}
             </span>
-            <span
+            <label
```

Comment:
Just make sure to test this on the Zapier side just in case anything is depending on this markup. Hopefully there's no evil `span` selector, but better to be safe. I guess it's murky whether this is a major release given it _technically_ has a breaking change here. But you could also call it a patch since you're fixing the accessibility issues. I'll leave that decision to you.

Reply:
I've never tested formatic in z/z, do we have a guide?  If not would you like to pair on that some time to make sure I'm not exploding anything?

Reply:
Primarily it's just a matter of `yarn link`, but yeah, happy to pair on more specifics!